src/components/30-organisms/modal/index.scss

```diff
@@ -0,0 +1,83 @@
+html,
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.o-modal {
+  display: none;
+  width: 100%;
+  height: 100%;
+
+  position: fixed;
+  z-index: 1;
+  left: 0;
+  top: 0;
+
+  background-color: rgba(0, 0, 0, 0.8);
+
+  &--open {
+    display: flex;
+    justify-content: center;
+    flex-flow: column;
+  }
+
+  &-window {
+    position: relative;
+    overflow: auto;
+    padding: 40px 40px 0px;
+    margin: 10px 10px 0px;
+    height: 100%;
+    background-color: white;
+
+    &__close-button {
+      position: absolute;
+      right: 18px;
+      top: 18px;
+      width: 24px;
+      height: 24px;
+    }
+
+    &__close-container {
+      display: flex;
+      justify-content: center;
+      padding: 30px 40px;
+      margin: 0px 10px 10px;
+      background-color: white;
+
+      &-button {
+        width: 100%;
+      }
+    }
+  }
+
+  &-close__axa-button {
+    margin: auto;
+  }
+}
+
+@include breakpoint($mediaquery-xs-up) {
+
+  .o-modal {
+    align-items: center;
+    &-window {
+      overflow: auto;
+      box-sizing: border-box;
+      padding: 50px;
+      width: 60%;
+      max-height: 50%;
+      border-radius: 4px;
+  
+      background-color: white;
```

Comment:
Remove the empty line 70. And maybe its better to user our color variables instead of "white" in the whole file. (not 100% sure about that)